docs/src/content/docs/drivers/postgres/getting-started.mdx

```diff
@@ -0,0 +1,216 @@
+
+## Usage examples
+
+<div data-no-swup>
+  <Tabs>
+    <TabItem label="Node.js">
+This example uses the [`postgres`](https://www.npmjs.com/package/postgres) package.
+
+```javascript
+"use strict"
+
+import postgres from 'postgres';
+
+const sql = postgres({
+  host: "localhost",
+  port: 5432,
+  fetch_types: false, // currently required to be 'false'
+  // debug: console.log
+});
+
+// Mapping of domain JavaScript types to Postgres OIDs
+const typeToOid = {
+  number: 23,  // Postgres OID for 'int4' (note that 20/'int8' will coerce to string by default due JS 53-bit numeric precision)
+  string: 25   // Postgres OID for 'text
+};
+// for the full list of OIDs see "Supported types" for the Postgres Driver at https://docs.xtdb.com
+
+function objsToTypedObjs(objs) {
+  return objs.map(o => {
+    const transformedEntry = {};
+    for (const key in o) {
+      if (o.hasOwnProperty(key)) {
+        const jsType = typeof o[key];
+        const oid = typeToOid[jsType];
+        if (oid) {
+          transformedEntry[key] = sql.typed(o[key], oid);
+        } else {
+          throw new Error(`Unsupported JS type: ${jsType} for key: ${key}`);
+        }
+      }
+    }
+    return transformedEntry;
+  });
+}
+
+async function insertTrades(trades) {
+  if (!Array.isArray(trades) || trades.length === 0) {
+    throw new Error('Trades must be a non-empty array');
+  }
+
+  return await sql`
+    INSERT INTO trades
+    ${ sql(objsToTypedObjs(trades)) }
+  `;}
+
+async function getTradesOver(quantity) {
+  return await sql`
+    SELECT *
+    FROM trades
+    WHERE quantity > ${ sql.typed(quantity, 23) }
+  `;}
+
+async function main() {
+  const trades = [
+    { _id: 1, name: 'Trade1', quantity: 1001 },
+    { _id: 2, name: 'Trade2', quantity: 15 },
+    { _id: 3, name: 'Trade3', quantity: 200 },
+  ];
+
+  try {
+    const tx = await insertTrades(trades);
+    console.log('Trades inserted successfully');
+  } catch (error) {
+    console.error('Error inserting trades:', error);
+  }
+
+  try {
+    const result = await getTradesOver(100);
+    console.log(result);
+  } catch (error) {
+    console.error('Error fetching trades:', error);
+  }
+  process.exit()
+}
+
+main();
+```
+    </TabItem>
+    <TabItem label="Python">
+This example uses the [Psycopg 3](https://www.psycopg.org/psycopg3/) database adapter.
+
+```python
+import asyncio
+import psycopg as pg
+
+DB_PARAMS = {
+    "host": "localhost",
+    "port": 5432
+}
+
+async def insert_trades(conn, trades):
+    if not isinstance(trades, list) or len(trades) == 0:
+        raise ValueError("Trades must be a non-empty list")
+
+    query = """
+    INSERT INTO trades (_id, name, quantity) VALUES (%s, %s, %s)
+    """
+
+    trade_values = [(trade["_id"], trade["name"], trade["quantity"]) for trade in trades]
+
+    async with conn.cursor() as cur:
+        await cur.executemany(query, trade_values)
```

Comment:
need to remove this and switch back to separate executes because it doesn't actually work